src/modules/layout/containers/__tests__/__snapshots__/LocationLayout.spec.js.snap

```diff
@@ -35,9 +35,23 @@ exports[`LocationLayout should show CategoriesToolbar if current route is catego
           },
         }
       }
+      cities={
+        Array [
+          CityModel {
+            "_code": "city1",
+            "_eventsEnabled": undefined,
+            "_extrasEnabled": undefined,
+            "_live": undefined,
+            "_name": "Mambo No. 5",
+            "_sortingName": undefined,
+          },
+        ]
+      }
       city="city1"
+      feedbackType={undefined}
```

Comment:
Quite some fields are undefined in this snap. You can try to add flow to fix this "automatically" :)

Reply:
But probably we are ok with this because we do not test for these values...